hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestDatanodeAdminMonitor.java

```diff
@@ -837,6 +837,54 @@ public void testCancelledNodesMovedToInService()
         nodeManager.getNodeStatus(dn1).getOperationalState());
   }
 
+  @Test
+  public void testContainersReplicatedOnDecomDnAPI()
+      throws NodeNotFoundException, ContainerNotFoundException {
+    conf.setBoolean("hdds.scm.replication.enable.legacy", true);
```

Comment:
Should be `false` since we use the new Replication Manager now. The default is also false.